test/unittests/execute_floating_point_test.cpp

```diff
@@ -100,9 +117,69 @@ struct WasmTypeName
     }
 };
 
+template <typename T>
+class execute_floating_point_types : public testing::Test
+{
+};
+
+template <typename T>
+bytes get_binop_code(Instr opcode)
+{
+    constexpr auto opcode_offset =
```

Comment:
Minor suggestion: I would just calculate result opcode here, perhaps it would be easier to understand, without guessing what this offset is.

Reply:
Done.